Scenes/Spaceship/Spaceship.gd

```diff
@@ -1,5 +1,7 @@
 extends KinematicBody2D
 
+signal gameOver()
```

Comment:
Little note: it would be nice to group signals (we have already have another signal in the script).